hooks/responsive.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+
```

Comment:
I'm seeing some type definition errors on this page. Would it be possible to add type definitions?